tutorials/light_ecModel/protocol.m

```diff
@@ -0,0 +1,138 @@
+% This file accompanies the GECKO3 Nature Protocols paper (DOI TO BE ADDED).
+%
+% The function of this script is to demonstrate the reconstruction and
+% analysis of a *light* ecModel. As example, it here uses the human-GEM
+% model of Homo sapiens as starting point. However, this script does not
+% claim to construct a "production-ready" ecHumanGEM model: dependent on
+% how you intend to use the ecModel it may require additional curation and
+% evaluation.
+%
+% DO NOT USE THE ECMODEL GENERATED HERE OUTSIDE OF THIS TUTORIAL.
+%
+% This script has limited comments and explanations of how the various
+% functions work, more detailed descriptions can be found in
+% tutorials/full_ecModel/protocol.m.
+
+% Prepare software and model adapter
+GECKOInstaller.install
+checkInstallation
```

Comment:
Stemming from [this comment](https://github.com/SysBioChalmers/GECKO/pull/319#discussion_r1239683063), `gurobi` should be used because the default `glpk` sees the problem in step as infeasible:
```
setRavenSolver('gurobi');
checkInstallation
```